parquet/src/main/java/org/apache/iceberg/parquet/ParquetIO.java

```diff
@@ -41,11 +41,11 @@
 /**
  * Methods in this class translate from the IO API to Parquet's IO API.
  */
-class ParquetIO {
+public class ParquetIO {
```

Comment:
Why do these methods need to be public? These should only be used to convert Iceberg's `InputFile` and `OutputFile` interfaces to versions that can be used with Parquet. Iceberg should keep these internal because all the external APIs should use Iceberg's interfaces.